packages/react-vapor/src/components/statusCard/styles/StatusCard.scss

```diff
@@ -21,6 +21,22 @@
     }
 }
 
+.isSuccess {
+    border-left-color: #2ba488; // var(--success-60)

```

Comment:
Why not use the variable?

Reply:
Also, I'm wondering if it would be better to create classes like `.mod-success` and scope them into `.statusCard`. In this way, we could try something like `border-left-color: var(--border-left-color);` then `--border-left-color: var(--success-60)` :thinking:

Reply:
@gdostie  :no_mouth: ... be right back :sweat_smile: 

@rosalie-liu  Mmmh :thinking: Where would you put the .mod ? In the Border.scss? or in the statusCard.scss? If we can get rid of the styles.something thingy to use the classes i'm all for it :eyes:
I just don't know how to do it, I'll need to look into it and learn :nerd_face: